lib/jbuild

```diff
@@ -1,7 +1,7 @@
 (library
  ((name functoria)
   (public_name functoria)
-  (libraries (unix cmdliner rresult fmt astring fpath))
+  (libraries (unix cmdliner rresult fmt astring fpath ptime.clock.os))
```

Comment:
not sure about best current practises: should `ptime` be included here as well, or is transitivity (via `ptime.clock.os`) sufficient?  I'm in favour of the former.

Reply:
in general I agree that transitive dependencies are a potential source of bugs, but within a single ocamlfind lib it's probably acceptable.